src/pymor/models/interface.py

```diff
@@ -349,6 +459,52 @@ def output(self, mu=None, return_error_estimate=False, **kwargs):
         else:
             return data['output']
 
+    def solve_d_mu(self, parameter, index, mu=None, **kwargs):
+        """Solve for the partial derivative of the solution w.r.t. a parameter index
+
+        Parameters
+        ----------
+        parameter
+            parameter for which to compute the sensitivity
+        index
+            parameter index for which to compute the sensitivity
+        mu
+            |Parameter value| for which to solve
+
+        Returns
+        -------
+        The sensitivity of the solution as a |VectorArray|.
+        """
+        data = self.compute(
+            solution_d_mu=(parameter, index),
+            mu=mu,
+            **kwargs
+        )
+        return data['solution_d_mu']
+
+    def output_d_mu(self, mu=None, return_array=False, **kwargs):
+        """compute the gradient w.r.t. the parameter of the output functional
+
+        Parameters
+        ----------
+        mu
+            |Parameter value| for which to compute the gradient
+        return_array
+            if `True`, returning a |NumPy array| and if `False`, returning
+            a dict w.r.t. to the parameters
```

Comment:
```suggestion
            If `True`, return the output gradient as a |Numpy array|. Otherwise, return a dict of gradients for
            each |Parameter|.
```